kms/awskms/signer.go

```diff
@@ -0,0 +1,122 @@
+package awskms
+
+import (
+	"crypto"
+	"crypto/ecdsa"
+	"crypto/rsa"
+	"io"
+
+	"github.com/aws/aws-sdk-go/service/kms"
+	"github.com/pkg/errors"
```

Comment:
In certificate and cli we use always that package that gives us the ability to create stacktraces of errors.

At some point, I'd like to deprecate it, because it doesn't support, and it will not support new error improvements that are present in the standard package since Go 1.13, but we will need to build the stacktrace support ourselves.

Reply:
Do we have a ticket for that? I could take a look at doing it.

Reply:
Probably not.

Reply:
https://github.com/smallstep/certificates/issues/278